applications/StructuralMechanicsApplication/custom_constitutive/small_strain_isotropic_damage_traction_only_3d.cpp

```diff
@@ -0,0 +1,84 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Marcelo Raschi
+//  Collaborators:
+//
+
+// Project includes
+#include "small_strain_isotropic_damage_traction_only_3d.h"
+#include "structural_mechanics_application_variables.h"
+#include "utilities/math_utils.h"
+
+namespace Kratos
+{
+//******************************CONSTRUCTOR*******************************************
+//************************************************************************************
+
+SmallStrainIsotropicDamageTractionOnly3D::SmallStrainIsotropicDamageTractionOnly3D()
+        : SmallStrainIsotropicDamage3D()
+{
+}
+
+//********************************COPY CONSTRUCTOR************************************
+//************************************************************************************
+
+SmallStrainIsotropicDamageTractionOnly3D::SmallStrainIsotropicDamageTractionOnly3D(
+        const SmallStrainIsotropicDamageTractionOnly3D &rOther) = default;
+
+//********************************CLONE***********************************************
+//************************************************************************************
+
+ConstitutiveLaw::Pointer SmallStrainIsotropicDamageTractionOnly3D::Clone() const
+{
+    return Kratos::make_shared<SmallStrainIsotropicDamageTractionOnly3D>(SmallStrainIsotropicDamageTractionOnly3D(*this));
+}
+
+//********************************DESTRUCTOR******************************************
+//************************************************************************************
+
+SmallStrainIsotropicDamageTractionOnly3D::~SmallStrainIsotropicDamageTractionOnly3D() = default;
+
+//************************************************************************************
+//************************************************************************************
+
+void SmallStrainIsotropicDamageTractionOnly3D::ComputePositiveStressVector(
+        Vector& rStressVectorPos, Vector& rStressVector)
+{
+    BoundedMatrix<double, 3, 3> stress_matrix;
+    BoundedMatrix<double, 3, 3> eigen_values(3, 3), eigen_vectors(3, 3);
```

Comment:
```suggestion
    BoundedMatrix<double, 3, 3> eigen_values, eigen_vectors;
```

Are bounded, you don't need that. And I think it will not work with AMatrix